ecr/Makefile

```diff
@@ -0,0 +1,31 @@
+all: init plan apply aws build tag push
+
+#Terraform
+
+init: 
+	terraform init
+
+plan: init
+	terraform plan 
+
+apply: plan 
+	terraform apply -auto-approve
+
+#AWS
+
+aws: apply 
```

Comment:
remove the apply from here.. this build/tag/push part is just a test to see if the service works, let's make it independent